src/views/v2/day/event.php

```diff
@@ -16,7 +16,7 @@
  * @see tribe_get_event() For the format of the event object.
  */
 
-$classes = get_post_class( [ 'tribe-common-g-row', 'tribe-common-g-row--gutters', 'tribe-events-calendar-day__event' ], $event->ID );
+$classes = tribe_get_post_class( [ 'tribe-common-g-row', 'tribe-common-g-row--gutters', 'tribe-events-calendar-day__event' ], $event->ID );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: c4a6301f2b4ac3f35673fe1e3db913c9</sub>

Reply:
Ignoring.